test/crowdsale/CapStagedCrowdsale.test.js

```diff
@@ -0,0 +1,69 @@
+const CapStagedCrowdsale = artifacts.require('CapStagedCrowdsale');
+const BigNumber = web3.BigNumber;
+const CapStagedCrowdsaleToken = artifacts.require('CapStagedCrowdsaleToken');
+require('chai').use(require('chai-as-promised')).use(require('chai-bignumber')(BigNumber)).should();
+
+contract('CapStagedCrowdsale', accounts => {
```

Comment:
please replace `accounts` with `[_, creator, investor]` and remove the lines 15 and 16

Reply:
Replaced accounts with function ([_, creator, wallet, investor])